test/Conversion/amd/ds_transpose.mlir

```diff
@@ -0,0 +1,77 @@
+// RUN: triton-opt %s --split-input-file --convert-triton-amdgpu-to-llvm=arch=gfx950 | FileCheck %s
+
+#mma16 = #ttg.amd_mfma<{versionMajor = 3, versionMinor = 0, warpsPerCTA = [2, 2], instrShape = [16, 16], isTransposed = true}>
+#mma32 = #ttg.amd_mfma<{versionMajor = 3, versionMinor = 0, warpsPerCTA = [2, 2], instrShape = [32, 32], isTransposed = true}>
```

Comment:
Does gfx950 have versionMajor = 4? Or is the change pending in another PR?

Reply:
Good point. It's added in https://github.com/triton-lang/triton/pull/5873. I just merged in `main`; tests might fail due to that.